### PR TITLE
New case of cache_sizes_test

### PR DESCRIPTION
**Pull request by hellohellenmao**
Test different cache sizes in a guest.

ID: 1814185
Signed-off-by: Tingting Mao <timao@redhat.com>

### CODE REVIEW

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.cache_sizes_test: PASS (404.22 s)

**Comment by hellohellenmao** [comment]
@lolyu @YongxueHong Help to take a review, please.

File: qemu/tests/cache_sizes_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    """
+    file = params["guest_file_name"]
+    initial_tag = params["images"]
+    cache_sizes = params["cache_sizes"].split()
+
+    logging.info("Boot a guest up from initial image: %s, and create a"
+                 " file %s on the disk.", initial_tag, file)
+    for cache_size in cache_sizes:
+        params["drv_extra_params"] = "cache-size=%s" % cache_size
+        base_qit = QemuImgTest(test, params, env, initial_tag)
```
Comment (lolyu) [inline qemu/tests/cache_sizes_test.py:26]: Please import from another case, in your case `QemuImgTest`

Comment (hellohellenmao) [reply qemu/tests/cache_sizes_test.py]: Updated, thanks. Review again, please.

**Comment by hellohellenmao** [comment]
(1/1) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.cache_sizes_test: PASS (415.90 s)

**Comment by hellohellenmao** [comment]
@YongxueHong Help to take a review, please

**Comment by hellohellenmao** [comment]
@lolyu, help to merge if it's okay to you, please.

**Comment by hellohellenmao** [comment]
@YongxueHong, help to merge if it's okay to you, please.

**Pull request merged**